src/site/resources/js/copy-clipboard.js

```diff
@@ -55,4 +55,11 @@ document.addEventListener("DOMContentLoaded", function () {
         });
     });
   });
-});
+};
+
+if (document.readyState === "loading") {
+    document.addEventListener("DOMContentLoaded", initCopyButtons);
+}
+else {
+    initCopyButtons();
+}
```

Comment:
Since the script uses the `DOMContentLoaded` event, we must do this `if-else` check to prevent a race condition. 

Just like in https://github.com/checkstyle/checkstyle/pull/18036#discussion_r2487706217. See that comment for more details.